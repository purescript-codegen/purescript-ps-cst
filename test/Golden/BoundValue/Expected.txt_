module BoundValue where

newtype Foo a
  = Foo a

foo :: Int
foo = x
  where
  Foo x = let Foo y = Foo (Foo 1) in y
