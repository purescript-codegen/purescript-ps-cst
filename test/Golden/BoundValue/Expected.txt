module BoundValue where

myfunc :: Int -> Int
myfunc 1 =
  let
    psModuleFile2 (Path x) = 1
   in
    psModuleFile 1
